bandit/src/query-lambda/bigquery.ts

```diff
@@ -21,7 +21,7 @@ export const getDataForBanditTest = async (
 	stage: 'CODE' | 'PROD',
 	test: Test,
 	date: Date = new Date(Date.now()),
-): Promise<{testName: string; rows: SimpleQueryRowsResponse}> => {
+): Promise<{testName: string;channel: string; rows: SimpleQueryRowsResponse}> => {
```

Comment:
it might be nice to make this a named type now - e.g.
`interface BigQueryResult { testName: string; ... }`